static/styles/portico/integrations_dev_panel.css

```diff
@@ -120,3 +119,9 @@
     margin-left: 2%;
     margin-right: 2%;
 }
+
+select,
+textarea,
+input {
+    font-family: inherit;
+}
```

Comment:
Could this still be inside `#dev-panel`? I think that'd be a lot cleaner if so, because then one can verify this will only affect that page locally, without needing to understand exactly in what places this file is included.

Reply:
(I guess also, does it make sense to add a comment that we can delete this once we've removed Bootstrap? I wonder if we wanted to use a specific format like `// TODO/bootstrap: This line will have no effect once Bootstrap is removed`)

Reply:
Updated the code to put this CSS inside `#dev-panel`. 

The user-agent `font-family` is not same (and are not inherited) for these elements, so we might need to keep this change even after removing the bootstrap code.